src/App.js

```diff
@@ -1,25 +1,29 @@
-import logo from './logo.svg';
+import React, { useState } from 'react';
 import './App.css';
+import Login from './Login'; 
+import mockUsers from '../src/mockusers'; 
 
-function App() {
+const App = () => { 
+  const [loggedInUser, setLoggedInUser] = useState(null);
+
+  const handleLogin = (username, password) => {
+    const user = mockUsers.find(
+      (user) => user.username === username && user.password === password
+    );
+    if (user) {
+      setLoggedInUser(user.username);
+    } else {
+      alert('Invalid username or password');
+    }
+  };
+  
   return (
     <div className="App">
       <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
+          <Login onLogin={handleLogin} />
```

Comment:
Login should have a path="/login" and be conditionally rendered if there is a user or not.

Reply:
!! i see how this would be a better way of organizing our routes.

Reply:
Implemented Routes to configure the different routes in the application.

<Route path="/" element={<Navigate to="/login" />} />: This <Route> element is used to handle the root URL ("/"). It uses the element prop to render the <Navigate> component and immediately redirects the user to the "/login" URL.

<Route path="/login" element={<Login onLogin={handleLogin} />} />: This <Route> element is used to handle the "/login" URL. When the user navigates to "/login", it renders the Login component, passing the handleLogin function as a prop to it